cli/command_restore.go

```diff
@@ -101,6 +103,8 @@ func addRestoreFlags(cmd *kingpin.CmdClause) {
 	cmd.Flag("skip-permissions", "Skip permissions during restore").BoolVar(&restoreSkipPermissions)
 	cmd.Flag("skip-times", "Skip times during restore").BoolVar(&restoreSkipTimes)
 	cmd.Flag("ignore-permission-errors", "Ignore permission errors").BoolVar(&restoreIgnorePermissionErrors)
+	cmd.Flag("ignore-errors", "Ignore all errors").BoolVar(&restoreIgnoreErrors)
+	cmd.Flag("incremental", "Incremental restore, do not download files that already exist in the destination").BoolVar(&restoreIncremental)
```

Comment:
Should this be called `skip-existing` or `skip-matching` to reflect what it does? Also, expand the flag comment to describe which types of entries it applies to (regular files and symlinks) and when are entries skipped and when they are not.

Reply:
Renamed to `--skip-existing`